plugins/workflow/process.go

```diff
@@ -200,18 +204,61 @@ func (wp *workflowProcess) handleCommand(id uint64, name string, params jsoniter
 
 	switch cmd := rawCmd.(type) {
 	case ExecuteActivity:
-		//acvitityID :=
-		wp.env.ExecuteActivity(cmd.ActivityParams(wp.env), wp.createCallback(id))
+		params := cmd.ActivityParams(wp.env)
+		activityID := wp.env.ExecuteActivity(params, wp.createCallback(id))
+
+		wp.canceller.register(id, func() error {
+			log.Print("cancel", activityID)
+			wp.env.RequestCancelActivity("activityID") // todo: wait for SDk fix
+			return nil
+		})
 
-		//wp.canceller.register(id, acvitityID)
+	case ExecuteLocalActivity:
+	// todo: param options are private
+	//localID := wp.env.ExecuteLocalActivity(&bindings.ExecuteLocalActivityOptions{}, wp.createCallback(id))
+	//wp.canceller.register(id, func() error {
+	//	log.Print("cancel", localID)
+	//	wp.env.RequestCancelActivity("activityID") // todo: wait for SDk fix
+	//	return nil
+	//})
 
-		// todo: local activity
-		// todo: child workflow
+	/**
+	 	ActivityID = make public (scheduleID, activityID)
+		LocalActivityID = make public (activityID)
+		TimerInfo(timerID string)
+	*/
 
-	case NewTimer:
-		//timerInfo := wp.env.NewTimer(cmd.ToDuration(), wp.createCallback(id))
+	//wp.env.RequestCancelActivity()
+
+	// todo: local activity
```

Comment:
Remove or why do we need skipped `case` logic here?

Reply:
Currenctly blocked by SDK.